packages/big-design/src/components/Chip/styled.tsx

```diff
@@ -0,0 +1,26 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+import { Box } from '../Box';
+import { StyleableButton } from '../Button/Button';
+
+export const StyledChip = styled(Box)`
+  align-items: center;
+  display: inline-flex;
+  user-select: none;
```

Comment:
🍹Is there a specific reason why we wouldn't want people to select the text?

Reply:
Not really, it just looks terrible and doesn't seem that useful 🤔